index.d.ts

```diff
@@ -22,7 +22,7 @@ type FastifyCorsHook =
   | 'onSend'
 
 declare namespace fastifyCors {
-  export type OriginFunction = (origin: string, callback: OriginCallback) => void;
+  export type OriginFunction = (origin?: string, callback: OriginCallback) => void;
```

Comment:
Do not use `?:` when the later argument is not optional.

```suggestion
  export type OriginFunction = (origin: string | undefined, callback: OriginCallback) => void;
```

Reply:
I wrote that first and then removed it to a `?` after seeing no other examples of `| undefined` used. Fully agree with this though! Thanks!